content/applications/finance/accounting/taxes/avatax/avatax_use.rst

```diff
@@ -43,8 +43,8 @@ Automatic triggers
 Manual triggers
 ---------------
 
-- :guilabel:`Compute taxes using Avatax` button at the bottom of the quote.
-- :guilabel:`Compute taxes using Avatax` button at the top of the invoice.
+- :guilabel:`Compute Taxes` button at the bottom of the quote.
+- :guilabel:`Compute Taxes` button at the top of the invoice.
```

Comment:
Do you think it would be beneficial to add some instructions to these explanations (e.g. "Click the...")... or do you think that is pretty self-explanatory? Just something to consider during your revisions. 🤔